.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict = true
```

Comment:
What does this do? Is it still needed after the update to min versions?

Reply:
OK I see, this is what caused the bail out to happen https://stackoverflow.com/questions/29349684/how-can-i-specify-the-required-node-js-version-in-package-json. Shame this has to be a separate file in the root of the project, but alright.

Reply:
If we omit this it won't enforce the min versions I believe.